core/src/main/kotlin/io/specmatic/test/DataRecorder.kt

```diff
@@ -0,0 +1,21 @@
+package io.specmatic.test
+
+import io.specmatic.core.log.HttpLogMessage
+
+object DataRecorder {
```

Comment:
Please rename to `TestInteractionsLog`

Reply:
Thanks, renamed it to `TestInteractionsLog`